src/tests/ftest/pool/pool_svc.py

```diff
@@ -130,6 +166,8 @@ def test_pool_svc(self):
                         "DaosServerManager.stop_ranks([{}])".format(
                             pool_leader))
 
+                self.is_pool_rebuild_in_progress()
```

Comment:
This function returns a bool, but seems you are not taking that into account here. How do you determine that something goes wrong if the number of retries (defaulted to 5) is reached and never changed the state to done?

Reply:
In case the rebuild is still in progress, deleting the pool will fail.

Reply:
Got it